Security/Authorization/Voter/UserAclVoter.php

```diff
@@ -31,21 +31,21 @@ public function supportsClass($class)
      */
     public function supportsAttribute($attribute)
     {
-        return $attribute === 'EDIT' || $attribute === 'DELETE';
+        return 'EDIT' === $attribute || 'DELETE' === $attribute;
     }
 
     /**
      * {@inheritdoc}
      */
-    public function vote(TokenInterface $token, $object, array $attributes)
+    public function vote(TokenInterface $token, $subject, array $attributes)
     {
-        if (!$this->supportsClass(get_class($object))) {
+        if (!is_object($subject) || !$this->supportsClass(get_class($subject))) {
             return self::ACCESS_ABSTAIN;
         }
 
         foreach ($attributes as $attribute) {
-            if ($this->supportsAttribute($attribute) && $object instanceof UserInterface && $token->getUser() instanceof UserInterface) {
-                if ($object->isSuperAdmin() && !$token->getUser()->isSuperAdmin()) {
+            if ($this->supportsAttribute($attribute) && $subject instanceof UserInterface && $token->getUser() instanceof UserInterface) {
```

Comment:
Please get rid of `$subject instanceof UserInterface`, this is already checked at line 42.

BTW, I'm pretty sure that this voter should deny when the token user is not an instance `UserInterface` and the `$subject` is a super admin. But this is not the purpose of this PR anyway.

Reply:
Could it be that `supportsClass` is overridden and support not only `UserInterface`?

Reply:
e.g. 
```
public function supportsClass($class)
{
    return is_subclass_of($class, 'FOS\UserBundle\Model\UserInterface') || is_subclass_of($class, 'MyClass');
}
```